src/nuxt.config.js

```diff
@@ -38,6 +38,7 @@ export default {
     '@nuxtjs/router-extras',
     '@nuxtjs/style-resources',
     '@nuxt/typescript-build',
+    '@nuxtjs/style-resources'
```

Comment:
This was already there (the second element); do we need it again?

Reply:
Nice catch - no clue why I did that